src/wasm-lib/kcl/tests/artifact_graph_sketch_on_face_etc/artifact_graph_flowchart.snap.md

```diff
@@ -124,14 +124,14 @@ flowchart LR
   26 --- 27
   26 --- 28
   26 --- 29
-  26 x--> 30
+  26 --- 30
```

Comment:
The sketch 26 used to be on a plane, but now the graph correctly has it connected bidirectionally with the end-cap 30 that it's on. So this is a good change.

Reply:
Oh dope, thanks for these comments, these mermaid diagrams are dope.

Reply:
You know that you can already see these in GitHub, right?

Reply:
No! I didn't :/

Reply:
Similar to SVG files, you can toggle between the markdown source and the rendered version. You can also click "View File" in the ellipsis menu.

![Screenshot 2025-02-11 at 1 29 56 PM](https://github.com/user-attachments/assets/f9fff797-a3a0-4d39-bd3e-d7f7f0b85992)

Reply:
The reason I was saying that I wish it were easier to create these is that you still need to commit the KCL and make a PR on GitHub. It's pretty easy in the grand scheme of things, but a lot of times, it's a one-off thing for debugging. nrc recently added a CLI for dev #5329. I might add it there.